recipes/boost-ext-ut/all/test_v1_package/CMakeLists.txt

```
@@ -1,14 +1,11 @@
-cmake_minimum_required(VERSION 3.15)
-project(test_package)
-
-set(CMAKE_CXX_STANDARD 20)
-set(CMAKE_CXX_STANDARD_REQUIRED yes)
-set(CMAKE_CXX_EXTENSIONS no)
+cmake_minimum_required(VERSION 3.8)
```

```suggestion
cmake_minimum_required(VERSION 3.12)
```
cxx_std_20 was added only on 3.12

https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html

---

Thanks!